tests/TestHelpers/SetupHelper.php

```diff
@@ -653,13 +465,7 @@ public static function readSkeletonFile(
 		);
 
 		//find the absolute path of the currently set skeletondirectory
-		$occResponse = self::runOcc(
-			['config:system:get', 'skeletondirectory'],
-			$xRequestId,
-			$adminUsername,
-			$adminPassword,
-			$baseUrl
-		);
+		$occResponse = ['code' => '', 'stdOut' => '', 'stdErr' => '' ];
```

Comment:
whole method can be removed. oCIS won't use `/apps/testing` endpoints

Reply:
sure, will do in next PR